src/features/user/repositories/user-repository/user-repository.test.ts

```diff
@@ -36,4 +38,49 @@ describe('[Repositories] UserRepository', () => {
       await expect(response).rejects.toThrowError();
     });
   });
+
+  describe('findById', () => {
+    it('should return user if found', async () => {
```

Comment:
It ... should acaba ficando redundante no ingles, pode evitar.

```suggestion
    it('return user if found', async () => {
```